Assets/Helpers/bevy_cronjob.toml

```diff
@@ -0,0 +1,3 @@
+name = "bevy_cronjob"
+description = "A simple helper for run cronjon in Bevy."
```

Comment:
```suggestion
description = "A simple helper to run cronjobs (at repeated schedule) in Bevy."
```

Reply:
a small explanation of cornjobs, everyone may not be familiar with what it is

Reply:
Note that this also fixes a typo and a grammar issue :)